shards/egui/src/widgets/text_field.rs

```diff
@@ -192,11 +206,19 @@ impl Shard for TextField {
       text_edit = text_edit.hint_text(RichText::from(hint).color(egui::Color32::GRAY).italics());
     }
 
-    text_edit = if !self.justify_width.get().is_none() && self.justify_width.get().try_into()? {
-      text_edit.desired_width(f32::INFINITY)
+    if let Ok(desired_width) = TryInto::<f64>::try_into(&self.desired_width.0) {
```

Comment:
Can you swap around, so if you set justify_width, it ignores desired width. It think that makes more sense, for example when you want to toggle that on and off while testing.